psrp/script.yaml

```
@@ -2036,7 +2036,7 @@
     Je vous en emballe une_?<end>
   pt-br: |
     Ei, um cliente! Que bom que você nos encontrou. Desculpe a localização.<wait more>
-    Meus bolos são famosos em Algol e é um roubo ser apenas <number><internal hint 02> meseta<s>. Quer um?<end>
+    Meus bolos são famosos em Algol, é um roubo ser apenas <number><internal hint 02> meseta<s>. Quer um?<end>
```

A period instead of a comma fits better here.
```suggestion
    Meus bolos são famosos em Algol. É um roubo ser apenas <number><internal hint 02> meseta<s>. Quer um?<end>
```

---

> A period instead of a comma fits better here.

Don't have É in script.

---

Understood. In that case you need to update [this line](https://github.com/ajkmetiuk/psrp/blob/f3d6348e5fd71f23922e65ac678482dd1fb6d6f8/psrp/script.yaml#L576) and [this line](https://github.com/ajkmetiuk/psrp/blob/f3d6348e5fd71f23922e65ac678482dd1fb6d6f8/psrp/script.yaml#L4798).